src/ml/neural_net/mps_layer_helper.mm

```diff
@@ -44,12 +45,15 @@ + (MPSCNNFullyConnectedNode *) createFullyConnected:(MPSNNImageNode *)inputNode
   MPSCNNFullyConnectedNode* fullyConnectedNode = 
     [MPSCNNFullyConnectedNode nodeWithSource:inputNode
                                      weights:fullyConnectedDataLoad];
+
+  fullyConnectedNode.weights = fullyConnectedDataLoad;
   
   return fullyConnectedNode;
 }
 @end
 
 @implementation MPSCNNConvolutionNode (TCMPSLayerHelper)
+@dynamic weights;
```

Comment:
`@dynamic` doesn't create the getter and setter, it just promises that one will be available at runtime. Where are these defined?

If not defined or used, please just remove the property for now, until you actually implement it.

Reply:
These are set in the helper methods `createConvolutional `, and accessed in the exportWeights method.

Reply:
But you haven't provided an implementation, so this will in fact be a run-time error